src/main/java/com/example/Back_end/Web/Services/ReservaService.java

```diff
@@ -0,0 +1,23 @@
+package com.example.Back_end.Web.Services;
+
+import com.example.Back_end.Web.DTOs.ReservaDTO;
+import com.example.Back_end.Web.Entities.Reserva.Reserva;
+import com.example.Back_end.Web.Repositories.ReservaRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ReservaService {
+    @Autowired
+    ReservaRepository reservaRepository;
+
+    public Reserva createReserva(ReservaDTO reservaData) {
```

Comment:
Vai ter que mexer aqui quando for implementar aquela logica do ManyToOne
Cria um User cliente que vai receber o UserRepository.findByID, Depois Cria a reserva e coloca
setCliende = cliente que tu criou. Nao sei se deu pra entender, ai dps tu salva no banco a reserva

Reply:
Pelo o que eu entendi eu salvaria a reserva do banco após trocar o ID_Cliente pelo User Cliente, contudo não teria que alterar a modelagem do banco?